datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/mlmodel/mappers/MLFeatureMapper.java

```diff
@@ -138,10 +139,12 @@ private static void mapMLFeatureKey(@Nonnull MLFeature mlFeature, @Nonnull DataM
   private static void mapMLFeatureProperties(
       @Nullable final QueryContext context,
       @Nonnull MLFeature mlFeature,
-      @Nonnull DataMap dataMap) {
+      @Nonnull DataMap dataMap,
+      @Nonnull Urn entityUrn) {
     MLFeatureProperties featureProperties = new MLFeatureProperties(dataMap);
-    mlFeature.setFeatureProperties(MLFeaturePropertiesMapper.map(context, featureProperties));
-    mlFeature.setProperties(MLFeaturePropertiesMapper.map(context, featureProperties));
+    var graphqlProperties = MLFeaturePropertiesMapper.map(context, featureProperties, entityUrn);
```

Comment:
`var`? let's use the actual type here - same with other files below.

or we can just keep in inline like it was before

Reply:
Are we against using `var`? It's one of those cases where we have the same name for the model class and graphql class so imo it's less readable than just using `var`